app-backup/burp/metadata.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+	<use>
```

Comment:
I'm not sure if I understand this. Why not just always-enable it as is the intention of the suggestion?

Reply:
you are right, of course... since the ipv6-USE-flag was introduced into the burp-ebuilds years ago, upstream has implemented an auto-detection into configure... 
I will remove the ipv6-USE-flag entirely with the next pull-request (burp-3.1.**_4_** has just been released anyway...)

Reply:
Ah, fantastic!